File: src/main/java/no/nav/domain/pensjon/regler/gomregning/KravFaktoromregningResultat.java
package no.nav.domain.pensjon.regler.gomregning;

import no.nav.domain.pensjon.regler.BatchStatus;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class KravFaktoromregningResultat implements Serializable {


    private Long kravId;
    private BatchStatus batchStatus;
    private List<PersongrunnlagOmregningResultat> persongrunnlagOmregningResultatListe;



}

File: src/main/kotlin/no/nav/pensjon/regler/domain/gomregning/KravFaktoromregningResultat.kt
package no.nav.pensjon.regler.domain.gomregning

import no.nav.pensjon.regler.domain.BatchStatus
import java.io.Serializable

class KravFaktoromregningResultat : Serializable {
    var kravId: Long? = null
    var batchStatus: BatchStatus? = null
    var persongrunnlagOmregningResultatListe: List<PersongrunnlagOmregningResultat> = mutableListOf()
}